### ISSUE TITLE
DAC 10 - High Priority WCAG Level A - Label in Name

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#10]
<img width="421" alt="DAC_Label_In_Name_01" src="https://user-images.githubusercontent.com/75850106/155552095-639db1a4-1e06-49fd-873b-45949af01c82.png">

### Issue
Voice activation users that speak the accessible name ‘Select a local authority’ for selecting a local authority can find that the select element does not respond. This is because the accessible name is not associated with the select element.

<img width="599" alt="DAC 10 Current code ref" src="https://user-images.githubusercontent.com/75850106/155552243-3cdb218d-14e6-4bc4-b773-384b579de562.png">

### Suggested Solution
Ensure that words which name an element are programmatically associated to the element that it names.
